ote/src/cljs/ote/views/main.cljs

```diff
@@ -93,8 +99,9 @@
                {:href     "#"
                 :on-click #(e! (fp-controller/->GoToUrl "/"))})
         [:img {:src "img/icons/nap-logo.svg" :style style-topnav/img }]]])
+
     (doall
-     (for [{:keys [page label url]} header-links]
+     (for [{:keys [page label url]} (header-links app)]
```

Comment:
Alternatively to the filter done in header links, you could do:
```
(for [{:keys [page label url]} (header-links app)
      :when page]
 ...body...
```

and take out the filtering

Reply:
Better to keep collections clean from nil values.